.gitignore

```diff
@@ -7,6 +7,9 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore vendor
+/vendor/gems
```

Comment:
I think: this is a nonstandard place for gems and usually you use `vendor` to explicitly _include_ dependencies (such as ones that aren't package managed).

Reply:
You're absolutely right, and inspired me to investigate what was going on and fix my local bundler config.